spec/inputs/syslog_spec.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
+require_relative "../spec_helper"
```

Comment:
so, for a 3 lines fix, you add an extra `require` and an extra file? plus all other fixes are in that file, let's stay consistent. we will have to revisit this soon anyway, no?